test/spoom/sorbet/version_test.rb

```diff
@@ -28,7 +28,7 @@ def test_return_nil_if_srb_not_installed
       def test_return_version_string
         Bundler.with_clean_env do
           version = Spoom::Sorbet.srb_version(path: "#{Cli::TestHelper::TEST_PROJECTS_PATH}/project")
-          assert_equal("0.5.5808", version)
+          assert_match(/\d\.\d\.\d{4}/, version)
```

Comment:
As to avoid changing the version every time in the test.

This test will fail if the version does not match `X.X.XXXX`, so it will fail if we cannot read the string and this is what we actually want to check.